docs/user-guide/docs-source/index.md

```diff
@@ -1,37 +1,13 @@
 
-Currently Ambit can be installed as a plugin for [Unreal Engine 4.27](https://docs.unrealengine.com/en-US/Support/Builds/ReleaseNotes/4_27/index.html).
+![Ambit sample image](images/AmbitBanner.jpg)
 
-## Getting Started
+Ambit is implemented as an Unreal Engine plugin and has been tested for compatibility with [Unreal Engine 4.27](https://docs.unrealengine.com/en-US/Support/Builds/ReleaseNotes/4_27/index.html).
 
-You can install the plugin either directly in the engine folder (in "Engine/Plugins/Editor/Ambit") or in your project folder (in "Plugins/Editor/Ambit").
 
-Make sure the Ambit plugin is installed and enabled in your Unreal Engine installation by going to **Edit** -> **Plugins** and ensuring the **enabled** checkbox for Ambit is checked.
 
-![Plugins Dropdown](./images/home/PluginDropdown.png)
-
-![Plugin Menu Enabled](./images/home/AmbitPlugin.png)
-
-## AWS Ambit Scenario Mode
-
-Once the plugin is enabled a new **AWS Ambit Scenario Mode** will be added to the **Modes** dropdown menu in the top panel. Select this Mode to enable scenario generation for your scene.
-
-![Modes Dropdown](./images/home/EnableAmbitMode_426.png)
-
-## Enabling Content
-
-Once the Ambit plugin is installed and enabled you will need to enabled plugin content in your content browser to access some of Ambit's tools. To do this, go to the **View Options** dropdown in the bottom right hand corner of your content browser and select **"Show Plugin Content"**.
-
-![Plugin Content Menu Enabled](./images/home/EnablePluginContent.png)
-
-Then "Ambit Content" folder should be selected in the "Sources Panel" of the Content Browser.
-
-![Source Panel Enabled](./images/home/SourcePanel.png)
-
-## Note
-
-During the usage of this plugin tool, Ambit may collect generic user metrics to inform us about the usage of this plugin tool. These metrics contain no identifiable info and are not shared or transferred.
-
-If you prefer to opt out from this metrics collection, within the Ambit plugin folder, you can edit the C++ header file Source/Ambit/Utils/UserMetricsSubsystem.h to make the RecordUserMetrics field false.
+> ✏️ **Note:** During the usage of this plugin tool Ambit will collect generic metrics to inform AWS about usage patterns and performance of its individual features. These metrics contain no personally identifiable information and are not shared or transferred.
+>
+> If you prefer to opt out from this metrics collection you can do so by making a small code modification and compiling the Ambit plugin from source. Within the Ambit plugin folder, edit the C++ header file `Source/Ambit/Utils/UserMetricsSubsystem.h` to make the `RecordUserMetrics` field false. Full details on how to compile Ambit from source can be found in ["Appendix: Contributing to Ambit Development"](appendix-contributing/).
```

Comment:
I think this should be a 'Disclaimer' rather than 'Note'

Also, why are these two different blocks?

Reply:
I will update with something more attention grabbing than "Note".

Regarding the two blocks, that's me fighting with the limitations of the way our doc parser generates HTML. If I stylize it so that a blockquote with multiple paragraphs looks like a single block, it looks great where there are mutliple paragraphs, but adds extra empty space at the end of blockquotes that contain a single sentence or paragraph. If I make single paragraph blockquotes look good, then I get the result you see in this case when there are multiple paragraphs. Since we have far more single paragraph blockquotes in our docs I decided to optimize for that case.

I might revisit the CSS later to see if I can come up with a better solution, but for now I'm gonna say it's good enough.